src/main/java/com/google/ehub/servlets/DashboardServlet.java

```diff
@@ -26,6 +26,7 @@
   private static final String ASCENDING_TITLE_SORT = "ASCENDING_TITLE";
   private static final String DESCENDING_TITLE_SORT = "DESCENDING_TITLE";
   private static final String RECENT_RELEASE_DATE_SORT = "RECENT_RELEASE_DATE";
+  private static final String HIGHEST_LIKES_SORT = "HIGHEST_LIKES";
```

Comment:
As an optimization, you can use an enum for the types of sorting available. Enums are type safe compared to string constants.